contracts/evm/xcall/script/CallService.s.sol

```diff
@@ -8,8 +8,14 @@ contract CallServiceScript is Script {
     function setUp() public {}
 
     function run() public {
-        vm.broadcast();
-        new CallService();
+        uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
+        vm.startBroadcast(deployerPrivateKey);
+
+        CallService xcall = new CallService();
+
+        xcall.initialize("bsc");
```

Comment:
@sdpisreddevil This should be the nid right? Do we want it hardcoded like this?

Reply:
No, this must be the NID. Should be taken from env too